modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/handlers/WebSocketRouter.java

```diff
@@ -249,6 +254,13 @@ private void connect() {
         }
     }
 
+    /**
+     * Connect to backend.
+     */
+    private void connect() {
+        connectorPool.submit(this::connect0);
```

Comment:
What problem we are trying to solve with connectorPool?

Reply:
Before fix connect will be executed from thread of websocket client that will be closed in connect.
Connect may be called twice from thread of client.